spark/v3.2/spark/src/main/java/org/apache/iceberg/spark/source/SparkMicroBatchStream.java

```diff
@@ -246,7 +269,7 @@ private void writeOffset(StreamingOffset offset, OutputFile file) {
         writer.flush();
       } catch (IOException ioException) {
         throw new UncheckedIOException(
-            String.format("Failed writing offset to: %s", initialOffsetLocation), ioException);
+                String.format("Failed writing offset to: %s", initialOffsetLocation), ioException);
```

Comment:
Please fix formatting. Looks like your settings are incorrect for continuation indents and that is introducing a lot of unnecessary changes, as well as incorrect style.

Reply:
Yes, my IDE java style indent is different from the default setting, just change it back to default and reformat the file. Thanks for catching it